apps/prairielearn/src/ee/pages/institutionAdminLti13/institutionAdminLti13.html.ts

```diff
@@ -0,0 +1,300 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { type Institution, type Lti13Instance } from '../../../lib/db-types';
+
+import { EncodedData } from '@prairielearn/browser-utils';
+import { compiledScriptTag } from '../../../lib/assets';
+
+export function InstitutionAdminLti13({
+  institution,
+  lti13Instances,
+  instance,
+  resLocals,
+  platform_defaults,
+}: {
+  institution: Institution;
+  lti13Instances: Lti13Instance[];
+  instance: Lti13Instance | null;
+  resLocals: Record<string, any>;
+  platform_defaults: Record<string, any>;
+}): string {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/head')%>", {
+          ...resLocals,
+          navPage: 'institution_admin',
+          pageTitle: 'LTI 1.3',
+        })}
+        ${compiledScriptTag('institutionAdminLti13.ts')}
+      </head>
+      <body>
+        ${renderEjs(__filename, "<%- include('../../../pages/partials/navbar') %>", {
+          ...resLocals,
+          institution,
+          navbarType: 'institution',
+          navPage: 'institution_admin',
+          navSubPage: 'lti13',
+        })}
+        <main class="container mb-4">
+          <h2 class="h4">LTI 1.3 / Learning Tools Interoperability</h2>
+          <p>${lti13Instances.length} instance${lti13Instances.length === 1 ? '' : 's'} configured.
+          </p>
+          <hr/>
+
+          <div class="row">
+            <div class="col-3">
+
+            ${lti13Instances.length > 0 ? 'Please select an instance:' : ''}
+
+            <nav class="nav nav-pills flex-column">
+              ${lti13Instances.map((i) => {
+                return html`
+                  <a class="nav-link ${i.id === instance?.id ? 'active' : ''}" href="${i.id}">
+                    <span style="white-space: nowrap"> ${i.name ? i.name : `#${i.id}`} </span>
+                    <span style="white-space: nowrap">(${i.platform})</span>
+                  </a>
+                `;
+              })}
+            </nav>
+            <form method="POST">
+              <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
+              <button class="btn btn-outline-success btn-block my-4" type="submit"
+                name="__action" value="add_instance">
+                Add a new LTI 1.3 instance
+              </button>
+            </form>
+          </div>
+
+          <div class="col-9">
+
+            ${displayLTI13Instance(instance, resLocals, platform_defaults)}
+
+          </div>
+        </main>
+      </body>
+    </html>
+  `.toString();
+}
+
+const displayLTI13Instance = function (instance, resLocals, platform_defaults) {
```

Comment:
Missing types. Conventionally, this kind of "component" (borrowing React nomenclature) is `PascalCase` and doesn't include "display" or any kind of verb. You can declare functions directly with `function LTI13Instance(...)` instead of using `const`.

Reply:
Addressed in a previous commit.